templates/web/pages/tag/tag.tt.html

```diff
@@ -48,6 +50,43 @@
             [% description %]
         </div>
     [% END %]
+    [% IF tagid.defined  %]
+        <!-- injecting facet-knowledge-panel -->
+        <div id="facet-knowledge-panel">
+            <h2 id="facet_panels_title"></h2>
+            <div id="facet_panels_content"></div>
+        </div>
+        <!-- Fetching facet knowledge panel -->
+        <script>
+            let facet_kp_url = "https://facets-kp.openfoodfacts.net/render-to-html?";
```

Comment:
we have to change the base url to be a parameter !

1. add it to .env
2. add it to variables passed to apache container (backend)
3. add PerlPassEnv for it in apache.conf
4. add it to Config2_docker.pm (reading from env) and export it (don't use `my` for declaration, it's global)
5. add it to your template data and handle the case where it's not declared, in this case do not use the script at all.